### PR TITLE
feat: create batch with local ssd

### PR DESCRIPTION
**Pull request by BigBlackWolf**
## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: batch/create/create_with_ssd.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    group = batch_v1.TaskGroup()
+    group.task_count = 4
+    group.task_spec = task
+
+    disk = batch_v1.AllocationPolicy.Disk()
+    disk.type_ = "local-ssd"
+    # The size of all the local SSDs in GB. Each local SSD is 375 GB,
+    # so this value must be a multiple of 375 GB.
+    # For example, for 2 local SSDs, set this value to 750 GB.
+    disk.size_gb = 375
```
Comment (m-strzelczyk) [inline batch/create/create_with_ssd.py:66]:
```suggestion
    disk.size_gb = 375
    assert(disk.size_gb % 375 == 0)
```
Let's make it even more obvious :)

File: batch/create/create_with_ssd.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+from google.cloud import batch_v1
+
+
+def create_local_ssd_job(
+    project_id: str, region: str, job_name: str, ssd_name: str
+) -> batch_v1.Job:
+    """
+    This method shows how to create a sample Batch Job that will run
+    a simple command on Cloud Compute instances with mounted local SSD.
+    Note: local SSD does not guarantee Local SSD data persistence.
```
Comment (m-strzelczyk) [inline batch/create/create_with_ssd.py:25]: It actually guarantees that it will not persist I think.

File: batch/create/create_with_ssd.py [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    volume.mount_path = f"/mnt/disks/{ssd_name}"
+    task.volumes = [volume]
+
+    # Tasks are grouped inside a job using TaskGroups.
+    # Currently, it's possible to have only one task group.
+    group = batch_v1.TaskGroup()
+    group.task_count = 4
+    group.task_spec = task
+
+    disk = batch_v1.AllocationPolicy.Disk()
```
Comment (m-strzelczyk) [inline batch/create/create_with_ssd.py:61]: I think you also need to pass the device name somewhere here for the disk object.

**Review by m-strzelczyk — CHANGES_REQUESTED** [review]

File: batch/create/create_with_ssd.py (hunk 3 above)
Comment (BigBlackWolf) [reply batch/create/create_with_ssd.py:61]: Yeah, it is passed, but _Disk_ doesn't have "name" property, it is in _AttachedDisk.device_name_ (line 78)

File: batch/create/create_with_ssd.py (hunk 2 above)
Comment (m-strzelczyk) [reply batch/create/create_with_ssd.py:25]: Let's leave it then.

File: batch/create/create_with_ssd.py (hunk 3 above)
Comment (m-strzelczyk) [reply batch/create/create_with_ssd.py:61]: Ah, I see now. You're right! :)

**Review by m-strzelczyk — APPROVED** [review]

**Pull request merged**